code/modules/uplink/uplink_item.dm

```diff
@@ -1176,6 +1176,13 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/weapon/storage/box/hug/reverse_revolver
 	restricted_roles = list("Clown")
 
+/datum/uplink_item/role_restricted/mimery
+	name = "Guide to Advanced Mimery Series"
+	desc = "The classical two part series on how to further hone your mime skills. Upon studying the series, the user should be able to make 3x1 invisble walls, and shoot bullets out of their fingers."
```

Comment:
You need to make it clear in the description they don't work if you aren't a mime job [because I think you can bypass job restrictions with agent IDs].